### DISCUSSION TITLE
Any package/plugin for support multiple lines in markdown.

### DISCUSSION THREAD

**Discussion by ShadmanAfzal** [remarkjs/remark#1181] [Q&A] [answered]
Hi All,

Can any please suggest some packages or plugin which can preserve multiple new lines while working with markdown?

For example.
1. Hello\n\n\n2. Hello

Should render as:
1. Hello <br><br><br>
2. Hello

But actually, it's rendering as:
1. Hello
2. Hello

Any help or suggestion is much appreciated.

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @ShadmanAfzal!

Collapsing newlines is expected behavior, there is not a plugin which keeps new lines as you describe.

See the past few times this has been asked for context and alternatives.
https://github.com/orgs/remarkjs/discussions/1095
https://github.com/orgs/remarkjs/discussions/993
